lib/airport.rb

```
@@ -0,0 +1,30 @@
+require './lib/plane'
+require './lib/weather'
+
+class Airport
+  attr_reader :planes, :capacity
+
```

You might not need to expose the attribute `capacity` to the outside with the accessor, as we probably don't want the caller/user of this class to access it directly?

You might ask: how do we test setting a custom capacity, then? Remember testing behaviour over state:
  * testing behaviour - given that we set a capacity of only 1, trying to land a plane *should raise an error*
  * testing state - given that we set a capacity of only 1, `airport.capacity` should return 1

Try to change your test below (in airport_spec, line 51) to assert on the class behaviour (it should raise an error if we go over capacity), rather than its state